prov/rxm/src/rxm_cq.c

```diff
@@ -1169,27 +1187,68 @@ static ssize_t rxm_atomic_send_resp(struct rxm_ep *rxm_ep,
 	return ret;
 }
 
-static void rxm_do_atomic(struct rxm_pkt *pkt, void *dst, void *src,
-			  void *cmp, void *res, size_t count,
-			  enum fi_datatype datatype, enum fi_op op)
+static int rxm_do_atomic(uint8_t op, void *dst, void *src, void *cmp, void *res,
```

Comment:
This function cannot fail, so should be void.  Don't adjust the call because it's convenient for the calling location.

Reply:
Will fix.